### PR TITLE
[PTE-12183]: Update jira issue description

### PR DESCRIPTION
**Pull request by emekafredy**
Jira link: https://syftapp.atlassian.net/browse/PTE-12183

### Why?
- platform and product_area from Jira issue description already exist as fields on the Jira tickets

### What?
- Remove platform and product_area from Jira issue description

### CODE REVIEW

**Comment by keram** [comment]
@emekafredy can you please submit PR to remove feature flag `bug_report_with_flex_domain_custom_field` first? 🙏🏻

**Comment by emekafredy** [comment]
@keram I just did this.
PR: https://github.com/Syft-Application/podpora-bot/pull/998

File: src/lib/support_config.ts [hunk 1]
```diff
@@ -187,7 +185,6 @@ Submitted by: ${user.name}`;
 Reason:
 ${submission.reason}
 
-Product Area: ${submission.product_area}
 Region/Country: ${submission.region}
```
Comment (kieran-flex) [inline src/lib/support_config.ts:188]: Region is already gone in the other PR isn't it?

Comment (kieran-flex) [reply src/lib/support_config.ts:188]: Ahh, have updated Jira ticket with my new understanding :)
https://syftapp.atlassian.net/browse/PTE-12182?focusedCommentId=92161

**Review by kieran-flex — APPROVED** [review]

**Pull request merged**